Doug/Items/Lootboxes/MysteryBox.cs

```diff
@@ -50,6 +50,7 @@ public MysteryBox(IInventoryRepository inventoryRepository, IRandomService rando
                 { new LootItem(HolyWater.ItemId, 1), 0.05 },
                 { new LootItem(SuicidePill.ItemId, 1), 0.05 },
                 { new LootItem(Cigarette.ItemId, 1), 0.05 },
+                { new LootItem(VapeMachine.ItemId, 1), 0.05 },
```

Comment:
balance this

Reply:
Make sure the total probability adds up to 1, it will still work otherwise but it's easier to calculate

Reply:
Done